webpack.config.js

```diff
@@ -38,7 +38,7 @@ module.exports = {
   },
   devtool: 'eval',
   output: {
-    libraryTarget: 'var',
+    libraryTarget: 'this',
     library: '[name]',
     path: './meinberlin/static/',
```

Comment:
just a side note: https://webpack.github.io/docs/configuration.html#output-path states this should be "The output directory as an *absolute path* (required)."